packages/core/src/tools/utils.ts

```diff
@@ -304,3 +307,52 @@ export function safeTruncate(candidate: string, length: number) {
   }
   return candidate.slice(0, length)
 }
+
+export function addGlobalEventListeners(
+  events: DOM_EVENT[],
+  listener: (event: Event) => void,
+  { once, capture }: { once?: boolean; capture?: boolean } = {}
+): { stop(): void } {
+  const wrapedListener = monitor(
+    once
+      ? (event: Event) => {
+          stop()
+          listener(event)
+        }
+      : listener
+  )
+
+  const options = supportPassiveEventListeners() ? { capture, passive: true } : capture
+  events.forEach((event) => addEventListener(event, wrapedListener, options))
+  const stop = () => events.forEach((event) => removeEventListener(event, wrapedListener, options))
+
+  return {
+    stop,
+  }
+}
+
+let supportPassiveEventListenersCache: boolean | undefined
+// Inspired by
+// https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Safely_detecting_option_support
+function supportPassiveEventListeners() {
```

Comment:
what is the behavior without this check?

Reply:
We have to have a check so event capture works on IE11, since it doesn't support addEventListener object options. Because we need to pass an option object only for passive events, I thought it would make sense to have this check.

Reply:
FMU, we did not need this test before because we did not have a use case where `{capture: false}` would be evaluated to a truthy capture value on IE11.
Now, with the wrapper we need to handle this case since we try to be passive everywhere.
is that is?

Reply:
Ah! I didn't know using an object would evaluate to true on IE11, that's great. OK to remove this check then. I see two alternatives:
* I don't think we need non-capturing event listeners, so we could simply remove the option and pass `{ capture: true, passive: true }` to all listeners. This makes the API easier to use.
* If you think that's a change a bit to risky, I can restore the previous behavior: add a `passive` option and if `true`, always use an option object, so listeners would be capturing on IE11 but not on newer browsers.

Reply:
> I didn't know using an object would evaluate to true on IE11

just a guess, I have not tested it

> OK to remove this check then

I am not saying that we should remove it, I am just trying to figure things out for now 🙂  

trying to be passive everywhere seems good to me, since it is not the default behavior for event listeners a comment on the function would be nice
if capturing everything would work for our current use cases, why not, it could worth ensuring it though.
I am not against keeping this check if we think we need or can have `capture: false` use cases.

Reply:
I think restoring the previous behavior (second alternative) would be safer for now, and we could try using only capturing listeners in a future PR.

Reply:
what make you choose this alternative over the current implementation?

Reply:
* We'd have the same behavior as in prod, giving us time to reconsider changing the behavior in a separate PR
* Shorter / simpler code

Reply:
> We'd have the same behavior as in prod

FMU, some events would be capturing in IE 11 but were not before, no?

Reply:
To be clear, I'd modify the function like this:
```ts
function addGlobalEventListeners(..., options: { once: boolean, capture: boolean, passive: boolean }) {
    [...]
    const options = passive ? { capture, passive } : capture
    [...]
}
```

The [only case](https://github.com/DataDog/browser-sdk/blob/5b4418b2446a02fc6b7c633b3af4a4da875bd30c/packages/core/src/domain/sessionManagement.ts#L146) of passive event listener we have is also capturing, so it doesn't change anything.

Reply:
ok clearer, lgtm